featurebyte/models/feature.py

```diff
@@ -242,6 +244,8 @@ class FeatureModel(BaseFeatureModel):
         Datetime when the Feature was first saved
     updated_at: Optional[datetime]
         When the Feature get updated
+    last_updated_at: Optional[datetime]
```

Comment:
There is another field `updated_at` in FeatureByteBaseDocumentModel. To avoid confusion, can we name this field to be more specific, something like `last_updated_by_scheduled_task_at`?